bigtable-client-core-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestBigtableOptionsFactory.java

```diff
@@ -86,7 +123,7 @@ public void testConnectionKeysAreUsed() throws IOException {
     configuration.set(BigtableOptionsFactory.BIGTABLE_HOST_KEY, "data-host");
     configuration.set(BigtableOptionsFactory.BIGTABLE_ADMIN_HOST_KEY, "admin-host");
     configuration.setInt(BigtableOptionsFactory.BIGTABLE_PORT_KEY, 1234);
-    configuration.setBoolean(BigtableOptionsFactory.BIGTABLE_USE_PLAINTEXT_NEGOTIATION, true);
+    configuration.setBoolean(BIGTABLE_USE_PLAINTEXT_NEGOTIATION, true);
     BigtableOptions options = BigtableOptionsFactory.fromConfiguration(configuration);
 
```

Comment:
static imports for BigtableOptionsFactory properties

Reply:
Thanks for the review!!... Statically imported constants.